src/l-marker-cluster-group.js

```diff
@@ -0,0 +1,63 @@
+// @ts-check
+import "leaflet.markercluster";
+import { layerConnected } from "./events.js";
+import LLayer from "./l-layer.js";
+
+class LMarkerClusterGroup extends LLayer {
+  static observedAttributes = [
+    "show-coverage-on-hover",
+    "icon-options"
+  ];
+
+  constructor() {
+    super();
+    this.layer = null;
+  }
+
+  connectedCallback() {
+    const name = this.getAttribute("name");
+    const iconOptions = this.getAttribute("icon-options");
```

Comment:
What happens if this is `null`?

Reply:
when `iconOptions` is nullish, `_createIconCreateFunction` will return undefined and leaflet.markercluster will use the default function instead. If the dev user code imported the leaflet.markercluster styles, it would present the default library clustering behaviour.